ecosystem/sep-0001.md

```diff
@@ -151,7 +151,51 @@ Replace `$PUBLIC_KEY` with the account's public key, `$CODE` with your asset cod
 
 Note: `fixed_number`, `max_number`, and `is_unlimited` are mutually exclusive issuance policies. Include exactly one of those fields.
 
-Alternately, `stellar.toml` can link out to a separate TOML file for each currency by specifying `toml="https://DOMAIN/.well-known/CURRENCY.toml"` as the currency's only field.
+
+### External Currency Specification
+There are two (compatible) ways to reference currency entries in separate files:
+
+  1. The `CURRENCIES` `toml` key.
+  2. The top-level `CURRENCY_TOMLS` list.
+
+#### `CURRENCIES` `toml` Key
+`stellar.toml` can link out to a separate TOML file for each currency by specifying the URL in the `toml` key as the currency's only field.
+
+Example usage:
+
+```toml
+[[CURRENCIES]]
+toml = "https://example.com/SPECIFIC_CURRENCY.toml"
+```
+
+#### `CURRENCY_TOMLS` List
+A top-level `CURRENCY_TOMLS` field may be specified as an array of URLs to TOML files that contain [currency entries](#currency-documentation). These additional fields are interpreted as if their contents exist in the `stellar.toml`. These files referenced must contain only `[[CURRENCIES]]` table entries.
+
+Example usage:
+
+```toml
+CURRENCY_TOMLS = [
```

Comment:
I think we should name the field `CURRENCIES_TOMLS` for an exact one-to-one correlation with the section that is allowed in the field. I think this was changed to `CURRENCY_TOMLS` to avoid a double plural, however this is a list of TOMLs, that each will contain multiple currencies, so I think the double plural here is not particularly problematic.

Reply:
Good point, it is a set of `TOMLS` each of which point to `CURRENCIES`. I hate the mouthful of a name (I'd love something more descriptive like `CURRENCY_METADATA_URLS` or something, [alas](https://martinfowler.com/bliki/TwoHardThings.html)), but it does do the job.